torchvision/models/feature_extraction.py

```diff
@@ -232,7 +233,9 @@ def get_graph_node_names(
             {"autowrap_modules": (math, torchvision.ops,),"leaf_modules": _get_leaf_modules_for_ops(),}
             WARNING: In case the user provides tracer_kwargs, above default arguments will be appended to the user
             provided dictionary.
-
+        concrete_args (Optional[Dict[str, any]]): Concrete arguments that should
+            not be treated as Proxies. This parameter is experimental and
+            its backwards-compatibility is *NOT* guaranteed.
```

Comment:
The original pytorch docs are confusing and also state https://pytorch.org/docs/stable/fx.html#torch.fx.Tracer.trace

> Backwards-compatibility for this API is guaranteed.


:thinking: